tests/lac/parallel_vector_08.cc

```diff
@@ -95,8 +106,12 @@ test()
 
   // now the same again, but import ghosts automatically because v had ghosts
   // set before calling operator =
-  v.reinit(local_owned, local_relevant, MPI_COMM_WORLD);
+  if (use_move)
+    v = std::move(original_v);
+  else
+    v.reinit(local_owned, local_relevant, MPI_COMM_WORLD);
```

Comment:
Don't you also have to do
```
  v = original_v;
```
here?

Reply:
I don't think so. In both move and non-move cases, we reset the vector to have the correct size, set it to ghost-mode, and then copy the values of w: at this point (before line 115) the actual values in the vector don't matter. Does this make sense? I think I'm following the original control flow of the test here.

Reply:
Ah, I didn't read this carefully enough.

What I would like is if in the `else` branch, you do all of these actions together: initialize, then import `w`, and if necessary update ghost values. In the `if` branch, this should all have happened as part of the move assignment already, right?

Reply:
I could, but - your questions are evidence that this is too messy. I rewrote the test to put the move assignment check in its own block.